person.rb

```diff
@@ -0,0 +1,26 @@
+# Blueprint for a person object
+class Person
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = rand(1..1000)
+    @age = age
+    @name = name
+    @parent_permission = parent_permission
+  end
+
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def can_use_services?
+    return true if is_of_age? || @parent_permission == true
```

Comment:
- [ ] There is a typo here. You are calling the wrong method `is_of_age`. I think you named as `of_age`. Please update it because I got an error when I try to test it.

See error screenshot:
![a1](https://user-images.githubusercontent.com/39523312/157327417-cfe05285-ee9b-45d7-98f8-23c74b11589a.png)

Reply:
Thanks! I fixed it